source/games/kalooki.html.md.erb

```diff
@@ -10,9 +10,28 @@ The objective of the game is to have the lowest score.
 
 Each game has 9 rounds. Each round requires players to collect a unique combination of sets and runs.
 
-A set is a collection of 3 or more cards of the same rank, disregarding suit.
+<%= partial "partials/hand", locals: {
+  caption: "A set is 3 or more cards of the same rank, disregarding suit.",
+  cards: [
+    "4 of diamonds",
+    "4 of hearts",
+    "4 of clubs"
```

Comment:
Tossing a trailing comma on this last line in the array is a nice touch to keep future diffs clean, if you ever add an item.